### PR TITLE
Add Anvil descriptor to top of readme

### PR DESCRIPTION
**Pull request by manggit**
Summary of Anvil and what Anvil does at the top of the Github repo

### CODE REVIEW

**Comment by manggit** [comment]
@simonetimen once you get a pass on this and the copy looks good, I'll replicate it across the remainder of our public repos

**Comment by simonetimen** [comment]
2 things: 
1) "generation" is misspelled where it says "PDF Generation".
2) "generation" and "filling" should both be lowercase.

**Comment by manggit** [comment]
@simonetimen can you give it the approval I need to merge?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+---
```
Comment (benogle) [inline README.md:7]: Need this HR?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+### Repository sponsored by [Anvil](www.useanvil.com/developers)
```
Comment (benogle) [inline README.md]: I think we should remove the 'sponsored by' text. It could just be 'What is Anvil?" or something like that

File: README.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+2. [PDF generation API](https://www.useanvil.com/products/pdf-generation-api/) - send markdown or HTML and Anvil will render it to a PDF
+3. [Etch E-sign with API](https://www.useanvil.com/products/etch/) - customizable, embeddable, e-signature platform with an API to control the signing process end-to-end
+4. [Anvil Workflows (w/ API)](https://www.useanvil.com/products/workflows/) - Webforms + PDF + E-sign with a powerful no-code builder. Easily collect structured data, generate PDFs, and request signatures. 
+
+Learn more on our [Anvil developer page](www.useanvil.com/developers)
+
+---
+
 # Load Balancing gRPC in Kubernetes
 This repoisoiry contains a configured gRPC sample application that can be run in Kubernetes to test proper load balancing.
```
Comment (benogle) [inline README.md]: Can you fix the repository typo?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
+---
+### Repository sponsored by [Anvil](www.useanvil.com/developers) 
+
+![Horizontal Lockupblack](https://user-images.githubusercontent.com/293079/169453889-ae211c6c-7634-4ccd-8ca9-8970c2621b6f.png#gh-light-mode-only)
+![Horizontal Lockup copywhite](https://user-images.githubusercontent.com/293079/169453892-895f637b-4633-4a14-b997-960c9e17579b.png#gh-dark-mode-only)
+
+Anvil provides easy APIs for all things paperwork.
+1. [PDF filling API](https://www.useanvil.com/products/pdf-filling-api/) - fill out a PDF template with a web request and structured JSON data
```
Comment (benogle) [inline README.md]: Periods at the end of each bullet? The last one has a period

File: README.md (hunk 2 above)
Comment (benogle) [reply README.md]: I'd also put this section below the first section of the readme, above the build section

Comment (manggit) [reply README.md]: I would like to keep the "sponsored by" section on all of our public repos that are not an Anvil client and not Anvil demo code. So SpectaQL, Microfiber etc. 

If we are going to market Anvil, we need to *market* Anvil.

Comment (manggit) [reply README.md]: Can move it down below the first section, do all of our repos follow this pattern in the readme?

File: README.md (hunk 1 above)
Comment (manggit) [reply README.md:7]: I think we should keep this especially if we are moving it below the first section - so it's clear its kind of separate from the readme.

File: README.md (hunk 4 above)
Comment (manggit) [reply README.md]: 👍

File: README.md (hunk 3 above)
Comment (manggit) [reply README.md]: 👍

**Review by simonetimen — APPROVED** [review]

**Pull request merged**